src/main/java/produto/TipoTamanho.java

```diff
@@ -1,8 +1,14 @@
 package produto;
 
 public enum TipoTamanho {
-    //IMPLEMENTE A LOGICA DO ENUM
-    //TODO
+    P,
+    M,
+    G;
+
     public final double multiplicador;
 
+    TipoTamanho() {
+        multiplicador = 0;
+    }
+
 }
```

Comment:
Exemplo de implementação do multiplicador no enum.
    
    public enum TipoTamanho {
        P (1.0), M(1.3), G(1.5);
        public final double multiplicador;

        TipoTamanho(final double multiplicador) {
            this.multiplicador = multiplicador;
        }
    }